### PR TITLE
Update faq.yml

### PR DESCRIPTION
**Pull request by dagdome**
new faq 1.2

## Description

As already mentioned in #ISSUE_NUMBER, this PR tackles:
* ...
* ...
* ...

In particular, the ...

## Checklist

- [ ] I followed the indications in the [CONTRIBUTING](CONTRIBUTING.md)
- [ ] The documentation related to the proposed change has been updated accordingly (also comments in code).
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?
- [ ] Ready for review! :rocket:

## Fixes

- Fixes #

### CODE REVIEW

**Comment by dagdome** [comment]
please @bfabio @denismarini

File: contents/faq-page/faq.yml [hunk 1]
```diff
@@ -452,7 +452,7 @@ questions:
           - comuni
           - scuole
       - title: È possibile erogare un servizio SaaS Qualificato AGID presso un CSP diverso da quello indicato nella scheda servizio pubblicata su marketplace?
-        content: In caso di cambio CSP sottostante ad un servizio SaaS, bisogna effettuare una nuova qualifica. Prerequisito è che il CSP di destinazione sia qualificato. La nuova qualifica deve essere tempestivamente inviata ad AGID che provvederà alla sua valutazione. La stessa include anche la conformità legislativa secondo cui il fornitore cloud deve rendere nota la localizzazione delle proprie infrastrutture e dove transiteranno i dati gestiti (e.g. IT, UE, Extra UE).
+        content: In caso di cambio CSP sottostante ad un servizio SaaS,bisogna effettuare una nuova qualifica. Prerequisito è che il CSP di destinazione sia qualificato. La nuova qualifica deve essere tempestivamente inviata ad AGID che provvederà alla sua valutazione. La stessa include anche la conformità legislativa secondo cui il fornitore cloud deve rendere nota la localizzazione delle proprie infrastrutture e dove transiteranno i dati gestiti (e.g. IT, UE, Extra UE).
```
Comment (bfabio) [inline contents/faq-page/faq.yml:455]:
```suggestion
        content: In caso di cambio CSP sottostante ad un servizio SaaS, bisogna effettuare una nuova qualifica. Prerequisito è che il CSP di destinazione sia qualificato. La nuova qualifica deve essere tempestivamente inviata ad AGID che provvederà alla sua valutazione. La stessa include anche la conformità legislativa secondo cui il fornitore cloud deve rendere nota la localizzazione delle proprie infrastrutture e dove transiteranno i dati gestiti (e.g. IT, UE, Extra UE).
```

File: contents/faq-page/faq.yml [hunk 2]
```diff
@@ -466,42 +466,54 @@ questions:
...
+        content: Il Comune potrà effettuare la migrazione avvalendosi dei due modelli di migrazione come delineato nella strategia nazionale per il cloud:<br>- Trasferimento in sicurezza dell’infrastruttura IT</br><br>- Aggiornamento in sicurezza di applicazioni in Cloud.</br>
         tag:
           - comuni
       - title: Un Comune è obbligato ad adottare la stessa modalità di migrazione per tutti i servizi?
-        content: No, il Comune può decidere singolarmente per ognuno dei servizi oggetto della migrazione. L’Ente potrà selezionare il modello di migrazione più adatto da presentare all’interno del piano di migrazione.
+        content: No,l’Ente potrà selezionare - singolarmente per ognuno dei servizi oggetto della migrazione - il modello di migrazione più adatto da presentare all’interno del piano di migrazione.
```
Comment (bfabio) [inline contents/faq-page/faq.yml:481]:
```suggestion
        content: No, l’Ente potrà selezionare - singolarmente per ognuno dei servizi oggetto della migrazione - il modello di migrazione più adatto da presentare all’interno del piano di migrazione.
```

File: contents/faq-page/faq.yml [hunk 3]
```diff
@@ -466,42 +466,54 @@ questions:
...
         tag:
           - comuni
       - title: Se ho già migrato un servizio in cloud in modalità "Trasferimento in sicurezza dell’infrastruttura IT" posso aderire al finanziamento per implementare la modalità "Aggiornamento in sicurezza di applicazioni in Cloud"?
-        content: Si, se il Comune ha già migrato un servizio in cloud in modalità "Trasferimento in sicurezza dell’infrastruttura IT" può accedere al finanziamento previsto dall'Avviso, riferito alla Misura 1.2, per l'implementazione della modalità "Aggiornamento in sicurezza di applicazioni in Cloud".In ogni caso, il processo di migrazione al cloud deve essere stato realizzato a decorrere dal 01 Febbraio 2020 con risorse finanziarie proprie.
+        content: Si,la Scuola può partecipare per l'implementazione della modalità "Aggiornamento in sicurezza di applicazioni in Cloud".
```
Comment (bfabio) [inline contents/faq-page/faq.yml:485]:
```suggestion
        content: Si, la Scuola può partecipare per l'implementazione della modalità "Aggiornamento in sicurezza di applicazioni in Cloud".
```

File: contents/faq-page/faq.yml [hunk 4]
```diff
@@ -466,42 +466,54 @@ questions:
...
         tag:
           - comuni
       - title: In cosa consiste la classificazione di dati e servizi? A cosa serve?
-        content: La classificazione dei dati e dei servizi gestiti dalle PA consente di uniformare e guidare il processo di migrazione al Cloud della PA. Le classi dei dati e servizi sono identificate sulla base del danno che una loro compromissione, in termini di confidenzialità, integrità e disponibilità, provocherebbe al sistema Paese.
+        content:La classificazione dei dati consiste nell'uniformare e guidare il processo di migrazione al cloud della PA. Le classi dei dati e servizi sono identificate sulla base del danno che una loro compromissione, in termini di confidenzialità, integrità e disponibilità, provocherebbe al sistema Paese. Tali classi sono: Strategico -Critico - Ordinario. L’applicazione del processo di classificazione permetterà un’analisi guidata degli impatti, nonché di eventuali requisiti di sicurezza e normativi, per l’identificazione dell’opportuna classe.
```
Comment (bfabio) [inline contents/faq-page/faq.yml:497]:
```suggestion
        content: La classificazione dei dati consiste nell'uniformare e guidare il processo di migrazione al cloud della PA. Le classi dei dati e servizi sono identificate sulla base del danno che una loro compromissione, in termini di confidenzialità, integrità e disponibilità, provocherebbe al sistema Paese. Tali classi sono: Strategico -Critico - Ordinario. L’applicazione del processo di classificazione permetterà un’analisi guidata degli impatti, nonché di eventuali requisiti di sicurezza e normativi, per l’identificazione dell’opportuna classe.
```

File: contents/faq-page/faq.yml [hunk 5]
```diff
@@ -515,39 +527,51 @@ questions:
...
+        content: No, il finanziamento concesso con il presente Avviso non è cumulabile con altri finanziamenti pubblici, nazionali, regionali o europei, per le stesse spese ammissibili, secondo quanto previsto dall'art. 8 comma 5 dell'Avviso.
         tag:
           - scuole
       - title: Di quante modalità di migrazione può avvalersi una scuola al fine di rispondere correttamente ai requisiti previsti dall'Avviso riferito alla Misura 1.2?
-        content: 'La Scuola potrà effettuare la migrazione sfruttando le diverse tipologie di migrazione delineate  nella Strategia Nazionale per il Cloud, selezionando tra le modalità:</br>- Trasferimento in sicurezza dell’infrastruttura IT</br>- Aggiornamento di applicazioni sicure in Cloud'
+        content:La Scuola potrà effettuare la migrazione sfruttando le diverse tipologie di migrazioni delineate nella strategia nazionale per il cloud, selezionando tra le modalità: <br>- Trasferimento in sicurezza dell’infrastruttura IT <br>- Aggiornamento di applicazioni sicure in Cloud
```
Comment (bfabio) [inline contents/faq-page/faq.yml:550]:
```suggestion
        content: La Scuola potrà effettuare la migrazione sfruttando le diverse tipologie di migrazioni delineate nella strategia nazionale per il cloud, selezionando tra le modalità: <br>- Trasferimento in sicurezza dell’infrastruttura IT <br>- Aggiornamento di applicazioni sicure in Cloud
```

File: contents/faq-page/faq.yml [hunk 6]
```diff
@@ -515,39 +527,51 @@ questions:
...
+        content:'La classificazione dei dati e dei servizi gestiti dalle PA consente di uniformare e guidare il processo di migrazione al Cloud della PA. Le classi dei dati e servizi sono identificate sulla base del danno che una loro compromissione, in termini di confidenzialità, integrità e disponibilità, provocherebbe al sistema Paese.<br><br>Ulteriori informazioni sono disponibili al seguente link: <a href="https://docs.italia.it/italia/cloud-italia/strategia-cloud-italia-docs/it/stabile/4_la_strategia_cloud_per_la_pubblica_amministrazione.html#la-classificazione-dei-dati-e-dei-servizi" aria-label="link esterno">https://docs.italia.it/italia/cloud-italia/strategia-cloud-italia-docs/it/stabile/4_la_strategia_cloud_per_la_pubblica_amministrazione.html#la-classificazione-dei-dati-e-dei-servizi</a>'
```
Comment (bfabio) [inline contents/faq-page/faq.yml:558]:
```suggestion
        content: 'La classificazione dei dati e dei servizi gestiti dalle PA consente di uniformare e guidare il processo di migrazione al Cloud della PA. Le classi dei dati e servizi sono identificate sulla base del danno che una loro compromissione, in termini di confidenzialità, integrità e disponibilità, provocherebbe al sistema Paese.<br><br>Ulteriori informazioni sono disponibili al seguente link: <a href="https://docs.italia.it/italia/cloud-italia/strategia-cloud-italia-docs/it/stabile/4_la_strategia_cloud_per_la_pubblica_amministrazione.html#la-classificazione-dei-dati-e-dei-servizi" aria-label="link esterno">https://docs.italia.it/italia/cloud-italia/strategia-cloud-italia-docs/it/stabile/4_la_strategia_cloud_per_la_pubblica_amministrazione.html#la-classificazione-dei-dati-e-dei-servizi</a>'
```

File: contents/faq-page/faq.yml [hunk 7]
```diff
@@ -466,42 +466,54 @@ questions:
...
         tag:
           - comuni
       - title: In cosa consiste la classificazione di dati e servizi? A cosa serve?
-        content: La classificazione dei dati e dei servizi gestiti dalle PA consente di uniformare e guidare il processo di migrazione al Cloud della PA. Le classi dei dati e servizi sono identificate sulla base del danno che una loro compromissione, in termini di confidenzialità, integrità e disponibilità, provocherebbe al sistema Paese.
+        content: La classificazione dei dati consiste nell'uniformare e guidare il processo di migrazione al cloud della PA. Le classi dei dati e servizi sono identificate sulla base del danno che una loro compromissione, in termini di confidenzialità, integrità e disponibilità, provocherebbe al sistema Paese. Tali classi sono: Strategico -Critico - Ordinario. L’applicazione del processo di classificazione permetterà un’analisi guidata degli impatti, nonché di eventuali requisiti di sicurezza e normativi, per l’identificazione dell’opportuna classe.
```
Comment (bfabio) [inline contents/faq-page/faq.yml:497]:
```suggestion
        content: >
        La classificazione dei dati consiste nell'uniformare e guidare il processo di migrazione al cloud della PA. Le classi dei dati e servizi sono identificate sulla base del danno che una loro compromissione, in termini di confidenzialità, integrità e disponibilità, provocherebbe al sistema Paese. Tali classi sono: Strategico -Critico - Ordinario. L’applicazione del processo di classificazione permetterà un’analisi guidata degli impatti, nonché di eventuali requisiti di sicurezza e normativi, per l’identificazione dell’opportuna classe.
```

File: contents/faq-page/faq.yml [hunk 8]
```diff
@@ -466,42 +466,55 @@ questions:
...
...ud nella Pubblica Amministrazione. La migrazione al Cloud permette alle pubbliche amministrazioni di fornire servizi digitali e di disporre di infrastrutture tecnologiche sicure, efficienti ed affidabili, in linea con i principi di tutela della privacy, con le raccomandazioni delle istituzioni europee e nazionali, mantenendo le necessarie garanzie di autonomia strategica del Paese, di sicurezza e controllo nazionale sui dati. In tale prospettiva, la strategia si muove lungo tre direttrici fondamentali: i) la creazione del Polo Strategico Nazionale (PSN), un’infrastruttura nazionale per l’erogazione di servizi Cloud, la cui gestione e controllo di indirizzo siano autonomi da fornitori extra UE, ii) un percorso di qualificazione dei fornitori di Cloud pubblico e dei loro servizi per garantire che le caratteristiche e i livelli di servizio dichiarati siano in linea con i requisiti necessari di sicurezza, affidabilità e rispetto delle normative rilevanti e iii) lo sviluppo di una metodologia di classificazione dei dati e dei servizi gestiti dalle pubbliche amministrazioni, per permettere una migrazione di questi verso la soluzione Cloud più opportuna (PSN o Cloud pubblico qualificato).
```
Comment (bfabio) [inline contents/faq-page/faq.yml:510]:
```suggestion
        content: > 
        La strategia per l'Italia digitale si sviluppa su due assi:digitalizzazione della PA e reti ultraveloci. La Strategia Cloud Italia è stata elaborata con l’obiettivo di fornire l’indirizzo strategico per l’implementazione e il controllo di soluzioni Cloud nella Pubblica Amministrazione. La migrazione al Cloud permette alle pubbliche amministrazioni di fornire servizi digitali e di disporre di infrastrutture tecnologiche sicure, efficienti ed affidabili, in linea con i principi di tutela della privacy, con le raccomandazioni delle istituzioni europee e nazionali, mantenendo le necessarie garanzie di autonomia strategica del Paese, di sicurezza e controllo nazionale sui dati. In tale prospettiva, la strategia si muove lungo tre direttrici fondamentali: i) la creazione del Polo Strategico Nazionale (PSN), un’infrastruttura nazionale per l’erogazione di servizi Cloud, la cui gestione e controllo di indirizzo siano autonomi da fornitori extra UE, ii) un percorso di qualificazione dei fornitori di Cloud pubblico e dei loro servizi per garantire che le caratteristiche e i livelli di servizio dichiarati siano in linea con i requisiti necessari di sicurezza, affidabilità e rispetto delle normative rilevanti e iii) lo sviluppo di una metodologia di classificazione dei dati e dei servizi gestiti dalle pubbliche amministrazioni, per permettere una migrazione di questi verso la soluzione Cloud più opportuna (PSN o Cloud pubblico qualificato).
```

File: contents/faq-page/faq.yml [hunk 9]
```diff
@@ -466,42 +466,55 @@ questions:
...
       - title: Come avviene l'acquisizione di servizi cloud qualificati?
-        content: I fornitori cloud che intendono erogare servizi Infrastructure as a Service (IaaS), Platform as a Service (PaaS) e Software as a Service (SaaS) destinati alle pubbliche amministrazioni devono ottenere, per questi servizi, la qualificazione rilasciata dall’Agenzia per l’Italia digitale (AgID), utilizzando la piattaforma dedicata alla qualificazione dei cloud service provider e dei servizi cloud.
+        content: I fornitori cloud che intendono erogare servizi Infrastructure as a Service (IaaS), Platform as a Service (PaaS) e Software as a Service (SaaS) destinati alle pubbliche amministrazioni devono ottenere, per questi servizi, la qualificazione rilasciata dall’Agenzia per l’Italia digitale (AGID), utilizzando la piattaforma dedicata alla qualificazione dei cloud service provider e dei servizi cloud. Ulteriori informazioni sono disponibili al seguente link: https://docs.italia.it/italia/cloud-italia/strategia-cloud-italia-docs/it/stabile/5_la_migrazione_della_pubblica_amministrazione_sul_cloud.html
```
Comment (bfabio) [inline contents/faq-page/faq.yml:517]:
```suggestion
        content: >
        I fornitori cloud che intendono erogare servizi Infrastructure as a Service (IaaS), Platform as a Service (PaaS) e Software as a Service (SaaS) destinati alle pubbliche amministrazioni devono ottenere, per questi servizi, la qualificazione rilasciata dall’Agenzia per l’Italia digitale (AGID), utilizzando la piattaforma dedicata alla qualificazione dei cloud service provider e dei servizi cloud. Ulteriori informazioni sono disponibili al seguente link: https://docs.italia.it/italia/cloud-italia/strategia-cloud-italia-docs/it/stabile/5_la_migrazione_della_pubblica_amministrazione_sul_cloud.html
```

File: contents/faq-page/faq.yml [hunk 10]
```diff
@@ -515,39 +528,51 @@ questions:
...
+        content: No, il finanziamento concesso con il presente Avviso non è cumulabile con altri finanziamenti pubblici, nazionali, regionali o europei, per le stesse spese ammissibili, secondo quanto previsto dall'art. 8 comma 5 dell'Avviso.
         tag:
           - scuole
       - title: Di quante modalità di migrazione può avvalersi una scuola al fine di rispondere correttamente ai requisiti previsti dall'Avviso riferito alla Misura 1.2?
-        content: 'La Scuola potrà effettuare la migrazione sfruttando le diverse tipologie di migrazione delineate  nella Strategia Nazionale per il Cloud, selezionando tra le modalità:</br>- Trasferimento in sicurezza dell’infrastruttura IT</br>- Aggiornamento di applicazioni sicure in Cloud'
+        content: La Scuola potrà effettuare la migrazione sfruttando le diverse tipologie di migrazioni delineate nella strategia nazionale per il cloud, selezionando tra le modalità: <br>- Trasferimento in sicurezza dell’infrastruttura IT <br>- Aggiornamento di applicazioni sicure in Cloud
```
Comment (bfabio) [inline contents/faq-page/faq.yml:551]:
```suggestion
        content: >
        La Scuola potrà effettuare la migrazione sfruttando le diverse tipologie di migrazioni delineate nella strategia nazionale per il cloud, selezionando tra le modalità: <br>- Trasferimento in sicurezza dell’infrastruttura IT <br>- Aggiornamento di applicazioni sicure in Cloud
```

**Pull request merged**